.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,43 +1,90 @@
 
-- [ ] My code follows the project's code style
-- [ ] I have commented my code where necessary
-- [ ] I have updated the documentation accordingly
-- [ ] My changes generate no new warnings
-- [ ] I have added tests that prove my fix/feature works
-- [ ] New and existing tests pass locally
-- [ ] I have checked my code for security issues
+<!-- File-grouped diff summary. One line per substantive file: "what changed and why", not "modified X". Skip pure formatting / import-only changes. If >15 substantive files, the PR is probably too big. -->
 
-## Screenshots (if applicable)
+**Backend:**
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)